docs/release-notes/next/PT-166698294-backchannel-posting.md

```diff
@@ -0,0 +1,3 @@
+* Off-chain updates can now still be completed even if one of the Websocket clients is off-line
```

Comment:
This info should also go somewhere into the module docs.

Reply:
Added some lines in the top-level comments of `aesc_fsm.erl`